transaction/transaction.go

```diff
@@ -17,7 +17,8 @@ import (
 
 type Transaction struct {
 	*pb.Transaction
-	hash *Uint256
+	hash                *Uint256
+	isSignatureVerified bool
```

Comment:
Make sure txn.isSignatureVerified would NOT be initialized automatic by Unmarshal() from malicious Node.Msg

Reply:
Both txn.hash and txn.isSignatureVerified will not be initialized. Only txn.Transaction will because it's a protobuf message